test/runtime/manifests/Policies-kafka-Role.json

```diff
@@ -3,7 +3,8 @@
 	"ingress": [{
 		"fromEndpoints": [
 			{"matchLabels":{"reserved:host":""}},
-			{"matchLabels":{"id.client":""}}
+			{"matchLabels":{"id.client":""}},
+			{"matchLabels":{"id.kafka":""}}
```

Comment:
This is not correct, the kafka -> zookeeper communication does not happen on port 9092 and the rule is a l3 +l4 rule

Reply:
Yes. My bad. Let me fix that

Reply:
fixed.